config/locales/en.yml

```diff
@@ -1,3 +1,7 @@
 en:
   service:
     name: Apply for criminal legal aid
+  time:
```

Comment:
I think instead of time what we want here is `date`. As you can have different formats for time in addition to date, for example:

```yaml
  time:
    formats:
      short: '%d %b %H:%M:%S'
```